Utilities/TestEvent/TestEvent.c

```diff
@@ -194,94 +207,111 @@ main (
-    printf ("PASS: SelfD event created\n");
+    DEBUG ((DEBUG_INFO, "PASS: SelfD event created\n"));
   }
 
-  // links events
-  // signal2 with signal3
+  //
+  // Link events
+  // Signal2 --> Signal3
+  // Wait1 --> Signal2
+  //
   SignalNotifyContext2.LinkedEvent = Signal3;
-  // wait1 with signal 2
-  WaitNotifyContext1.LinkedEvent = Signal2;
+  WaitNotifyContext1.LinkedEvent   = Signal2;
 
-  // try sets timers
+  //
+  // Try to set timers
+  //
   Status = gBS->SetTimer (Wait1, TimerPeriodic, 100*10000);
 
   if (EFI_ERROR (Status)) {
-    printf ("FAIL: Set timer for Wait1 (Status: %lu)\n", Status);
+    DEBUG ((DEBUG_INFO, "FAIL: Set timer for Wait1 (Status: %r)\n", Status));
   } else {
-    printf ("PASS: Wait1 set to 100*10000 periodic\n");
+    DEBUG ((DEBUG_INFO, "PASS: Wait1 set to 100*10000 periodic\n"));
   }
 
   Status = gBS->SetTimer (Timer1, TimerPeriodic, 10*10000);
 
   if (EFI_ERROR (Status)) {
-    printf ("FAIL: Set timer for Timer1 (Status: %lu)\n", Status);
+    DEBUG ((DEBUG_INFO, "FAIL: Set timer for Timer1 (Status: %r)\n", Status));
   } else {
-    printf ("PASS: Timer1 set to 10*10000 periodic\n");
+    DEBUG ((DEBUG_INFO, "PASS: Timer1 set to 10*10000 periodic\n"));
   }
 
   Status = gBS->SetTimer (SelfD1, TimerRelative, 30*10000);
 
   if (EFI_ERROR (Status)) {
-    printf ("FAIL: Set timer for SelfD1 (Status: %lu)\n", Status);
+    DEBUG ((DEBUG_INFO, "FAIL: Set timer for SelfD1 (Status: %r)\n", Status));
   } else {
-    printf ("PASS: SelfD1 set to 30*10000 relative\n");
+    DEBUG ((DEBUG_INFO, "PASS: SelfD1 set to 30*10000 relative\n"));
   }
 
-  // try emit signal1
+  //
+  // Try to emit Signal1
+  //
   gBS->SignalEvent (Signal1);
 
-  // try wait events
-  EFI_EVENT  Events[] = { Timer1, Wait1 };
+  //
+  // Try to wait events
+  //
+  Events[0] = Timer1;
+  Events[1] = Wait1;
 
-  for (int i = 0; i < 10; i++) {
+  for (Jndex = 0; Jndex < 10; Jndex++) {
     Status = gBS->WaitForEvent (2, Events, &Index);
     if (!EFI_ERROR (Status)) {
-      printf ("Event %lu triggered\n", Index);
+      DEBUG ((DEBUG_INFO, "Event %u triggered\n", Index));
       if (Index == 0) {
-        printf ("PASS: Timer1 event wait succeeded\n");
+        DEBUG ((DEBUG_INFO, "PASS: Timer1 event wait succeeded\n"));
         TimerCounter1++;
       }
     } else {
-      printf ("Event Error %lu (Status: %lu)\n", Index, Status);
+      DEBUG ((DEBUG_INFO, "Event Error %u (Status: %r)\n", Index, Status));
     }
   }
 
-  // try check event
+  //
+  // Try to check event
+  //
   Status = gBS->CheckEvent (Signal1);
   if (Status == EFI_SUCCESS) {
-    printf ("FAIL: Event check succeeded\n");
+    DEBUG ((DEBUG_INFO, "FAIL: Event check succeeded\n"));
   } else {
-    printf ("PASS: Event check error (Status: %lu)\n", Status);
+    DEBUG ((DEBUG_INFO, "PASS: Event check error (Status: %r)\n", Status));
   }
 
-  // TPL Test
+  //
+  // The TPL test
+  //
   EFI_TPL  OriginalTPL = gBS->RaiseTPL (TPL_HIGH_LEVEL);
```

Comment:
Split declaration and assignment. Place declaration at the beginning of the function.